<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="layout_CentralWidget">
    <item row="0" column="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Footlight MT Light</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:1, stop:0.517413 rgba(0, 85, 127, 255));
border-color: rgb(255, 255, 0);
color: rgb(255, 255, 255);
border-style:outset;
border-width:4;
border-radius:10px;

</string>
      </property>
      <property name="text">
       <string>Taskhir AutoGallery</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_backPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_CurrentDateTime">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active/>
        <inactive/>
        <disabled/>
       </palette>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_Wellcome">
       <layout class="QGridLayout" name="layout_WelcomePage">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_SignLog_in">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:1, stop:0.517413 rgba(0, 85, 127, 255));
border-color: rgb(255, 255, 0);
color: rgb(255, 255, 255);
border-style:outset;
border-width:4;
</string>
          </property>
          <property name="text">
           <string>Log(Sign) In</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget_Vitrine">
          <item>
           <property name="text">
            <string>ماشین 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ماشین 2</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_Home">
       <layout class="QGridLayout" name="layout_HomePage">
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_Finance">
          <property name="text">
           <string>Finance</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_Cars">
          <property name="text">
           <string>Cars</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableWidget_important_people">
          <column>
           <property name="text">
            <string>نام و نام خانوادگی</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>شماره تلفن</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>مبلغ مورد نظر</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="Page_Cars" name="page_Cars"/>
      <widget class="QWidget" name="page_Finance">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <layout class="QGridLayout" name="layout_FinancePage">
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_addPoorsant">
          <property name="text">
           <string>اضافه کردن درآمد</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_addNewCheck">
          <property name="text">
           <string>اضافه کردن چک جدید</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="3">
         <widget class="QWidget" name="widget_ChartPlaceHolder" native="true"/>
        </item>
        <item row="0" column="0" rowspan="3">
         <widget class="QTableWidget" name="tableWidget">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>تاریخ</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>مبلغ</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>بانک</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>شعبه</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>شناسه</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCalculator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionCalculator">
   <property name="text">
    <string>Calculator</string>
   </property>
  </action>
  <action name="actionSignIn">
   <property name="text">
    <string>SignIn</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Page_Cars</class>
   <extends>QWidget</extends>
   <header>Page_Cars.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Cars</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>navigateToCarsPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Finance</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>navigateToFinancePage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_addNewCheck</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addNewCheck()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_addPoorsant</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addIncome()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>navigateToCarsPage()</slot>
  <slot>navigateToFinancePage()</slot>
  <slot>addNewCheck()</slot>
  <slot>addIncome()</slot>
 </slots>
</ui>
